Services/CO.CDP.EntityVerification/Persistence/IDatabasePponRepository.cs

```diff
@@ -0,0 +1,12 @@
+namespace CO.CDP.EntityVerification.Persistence;
+
+public interface IDatabasePponRepository : IDisposable
```

Comment:
Should be named `IPponRepository`. Interfaces should not be design with a specific implementation in mind, as it defeats their purpose of providing a layer of decoupling.

Reply:
Done.